casepro/backend/junebug.py

```diff
@@ -40,6 +46,60 @@ def get_addresses(self, uuid):
         return (
             a['address'] for a in addresses if a.get('address') is not None)
 
+    def get_identities(self, **kwargs):
+        '''Get the list of identities filtered by the given kwargs.'''
+        url = '%s/api/v1/identities/search/?' % (self.base_url)
+        for key, value in kwargs.iteritems():
+            url = '%s%s=%s&' % (url, key, value)
```

Comment:
Maybe we should use the `params` dictionary below instead of manually doing it ourselves here? That should take care of properly encoding the params (if there are any special characters). Also, I don't think we need the `default=True` parameter here, I think that only applies to the `/addresses/` endpoint.